docs/ABI/RegisterAllocation.md

```diff
@@ -0,0 +1,13 @@
+## 64 bit Architecture Register Allocations
+
+From Swift 5, the calling convention register allocation for 64 architectures is largely based on [existing](https://developer.apple.com/library/content/documentation/Xcode/Conceptual/iPhoneOSABIReference/Articles/ARM64FunctionCallingConventions.html) [standards](https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/LowLevelABI/140-x86-64_Function_Calling_Conventions/x86_64.html) with the addition of the context and error return registers discussed in the ABI manifesto:
+
+| Register Purpose | ARM64 | x86_64 |
+| ------------- |:-------------:| ----- |
+| Context register (self) | x20 | r13 |
+| Error return register | x21 | r12 |
+| Struct return pointer | x8 | rax |
+| Float call arguments | d0 … d7 | xmm0 … xmm7 |
```

Comment:
d0 ... d7 isn't quite right for float arguments (or returns); single-precision is passed in s0 ... s7, half would be passed in h0 ... h7, and HFAs get passed in q0 ... q7. Simply saying `v0 ... v7` is probably the most correct option here.

Reply:
Thanks, fixed.

Reply:
Hi @stephentyrone, Can you reset your review?